README.md

```diff
@@ -1,9 +1,9 @@
-# Liberty Dev Mode for VS Code
-A VS Code extension for Liberty dev mode. The extension will detect your Liberty Maven project if it detects the `io.openliberty.tools:liberty-maven-plugin` in the `pom.xml`. Through the Liberty Dev Dashboard explorer on the side bar, you can start, stop, or interact with dev mode on all available Liberty dev projects in your workspace.
+# Open Liberty Tools for VS Code
+A VS Code extension for Open Liberty. The extension will detect your Liberty Maven project if it detects the `io.openliberty.tools:liberty-maven-plugin` in the `pom.xml`. Through the Liberty Dev Dashboard explorer on the side bar, you can start, stop, or interact with Liberty dev mode on all available Liberty Maven projects in your workspace.
```

Comment:
`Liberty dev mode` could link to the dev mode doc at https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#dev

Reply:
Created a new pull request to incorporate this change: https://github.com/OpenLiberty/liberty-dev-vscode-ext/pull/16